bitnami/harbor/templates/notary/notary-secret-envvars.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.notary.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "harbor.notary-server" . }}-envvars
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/component: notary
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
+  server_db_url: {{ include "harbor.database.notaryServer" . }}
```

```suggestion
  server_db_url: {{ include "harbor.database.notaryServer" . | b64enc | quote }}
```

We could add a bit of encryption to the data by default.

Also, please add a newline at the end of the file.